server/log/router.ts

```diff
@@ -14,8 +21,51 @@ export const LogModule: UIServerModule = {
     const routerForModule = express.Router();
 
     // implementation to follow
-    routerForModule.get('*', authFn, (req, res) => res.sendStatus(200));
+    routerForModule.get('*', authFn, (req, res) => {
+      const { isWs } = req as strimziUIRequestType;
+      const { ws } = res as strimziUIResponseType;
+      if (isWs) {
+        ws.on('message', handleMessage(logger));
+        ws.on('close', handleClose(logger));
+      } else {
+        // Return 426 Upgrade Required if this isn't a websocket request
```

Comment:
Is this just belts and braces on the error case? Could return `404` as it's not a valid call for non web?

Reply:
We expect HTTP GET calls to the `/log` endpoint to be websocket upgrade requests. If we get a request that is not an upgrade, then we return the [426](https://httpstatuses.com/426) response, as we don't serve any other http traffic from `/log`.